src/controllers/hid/hidcontroller.cpp

```diff
@@ -241,23 +214,12 @@ int HidController::open() {
         return -1;
     }
 
+    // Set hid controller to non-blocking
+    hid_set_nonblocking(m_pHidDevice, 1);
```

Comment:
Don't forget to handle the result of each API call!

Reply:
Looks like this function always returns 0 on all platforms, but yeah, probably a good thing to at least throw a warning.

Reply:
The API docs say 0 on success and -1 on error. Use VERIFY_OR_DEBUG_ASSERT and let open() fail with a warning log. An error is unexpected but we cannot be sure.